internal/services/sources_service.go

```diff
@@ -9,16 +9,31 @@ import (
 )
 
 func ListSources(w http.ResponseWriter, r *http.Request) {
+	var sourcesList []*clients.Source
+
 	client, err := clients.GetSourcesClient(r.Context())
 	if err != nil {
 		renderError(w, r, payloads.NewClientError(r.Context(), err))
 		return
 	}
-	sourcesList, err := client.ListProvisioningSources(r.Context())
-	if err != nil {
-		renderError(w, r, payloads.NewClientError(r.Context(), err))
+	hyperscaler := r.URL.Query().Get("hyperscaler")
```

Comment:
We already use "Provider" term in multiple places in our OpenAPI, let’s not introduce another term for the same thing.

```suggestion
	hyperscaler := r.URL.Query().Get("provider")
```